js/node/node-backend.js

```diff
@@ -13,6 +13,10 @@ const wasm = wasm_;
 
 export { wasm, withThreadPool };
 
+export async function initO1() {
+  // TODO: what needs to be done here?
+}
+
```

Comment:
I think we don't need this method (there's no async setup needed in this file), the fork point between Node and Web exports is `snarky.js` vs `snarky.web.js`.

There is a TLA in `snarky.js` which should probably be included in the init function (but there's also many caveats to that.. I mean why should the Node.js build even be bundled in the first place? And TLA really shouldn't be an issue in a Node.js package.. And the one place where we use TLA there is just to workaround a Jest weirdness and we want to get rid of Jest 😬 )